t/t2315-resource-system.t

```diff
@@ -101,4 +101,20 @@ test_expect_success 'invalid exclude hosts cause instance failure' '
     grep "nosuchhost: Invalid argument" ${name}/logfile
```

Comment:
Commit message: "doesn't not" - extra negative there?

Reply:
Ok, will fix to "doesn't not not" 😉